app/controllers/Cluster.scala

```diff
@@ -114,6 +122,8 @@ class Cluster (val messagesApi: MessagesApi, val kafkaManagerContext: KafkaManag
         )(ClusterTuning.apply)(ClusterTuning.unapply)
       )
       , "securityProtocol" -> nonEmptyText.verifying(validateSecurityProtocol)
+      , "saslMechanism" -> nonEmptyText.verifying(validateSASLmechanism)
```

Comment:
why is this field required?  If I am not using SASL, then why would we want the user to fill this in.